C/c4Query.cc

```diff
@@ -60,10 +60,13 @@ Retained<C4Query> C4Query::newQuery(C4Collection *coll, C4QueryLanguage language
     } catch (Query::parseError &x) {
         if (outErrorPos) {
             *outErrorPos = x.errorPosition;
-            return nullptr;
```

Comment:
Was this removal on purpose?

Reply:
yes, because I don't think we should change the logic here merely because being provided outErrorPos. The exception may contain other useful info, for example, why the syntax error, that the platform may deicide whatever to do with it.